### ISSUE TITLE
travis is not deploying linux nightlies

### ISSUE THREAD

**Issue by a-teammate** [inexorgame/inexor-core#437]
Actually in the [GitHub/releases](https://github.com/inexorgame/inexor-core/releases) page there should be a Inexor-0.8.x-Linux.zip as well, but Travis skips uploading.

Current error is that the repo is wrongly named, so it skips deployment.

No idea what the next error will be, but before we renamed the repo it was not uploading either (without error message at all).
I changed some really small stuff in the maintime, so dunno if that fixed it.
(Check the history for the travis fails and the recent travis fixes)

**Comment by a-teammate** [comment]
**Update**

The error messages are just not existent.

It does upload the `Inexor-0.8.9-alpha-doc.zip` using travis. it does not upload the nightlies.

This is the job which should deploy the package:
https://travis-ci.org/inexorgame/inexor-core/jobs/253590081
it does not. but it tells us:

```
Logged in as Inexor Team's Bot
Deploying to repo: inexorgame/inexor-core
Current tag is: 0.8.9-alpha

Preparing deploy

Deploying application

Done. Your build exited with 0.
```
yeah! err.. wait? no.. -.-

@Fohlen: you know travis ssh debugging, plz do this.

**Comment by Fohlen** [comment]
I will dig into that.
For the future, https://docs.travis-ci.com/user/running-build-in-debug-mode/

**Comment by a-teammate** [comment]
Solved with 2078d8b7e44a3100f80827add58f965f614f3bf0.
See https://github.com/inexorgame/inexor-core/releases/tag/0.8.10-alpha

Notice how the two packages are just built for 64 bit (but named differently; we could change that in the future I believe)
